.idea/runConfigurations/general_set_url_on_input_artifact.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="covid.general.set_url_on_input_artifact" type="PythonConfigurationType" factoryName="Python" folderName="covid">
```

Comment:
@kleppari Are you sure this is correct? Says covid on all the other configs here, so I'm wondering what is correct. We can change it later though...